manning/manning-proposal-2020-06.md

```diff
@@ -0,0 +1,204 @@
+2.  Use branches, pull requests, and code review to add a feature to the Python programs invoked by that script.
+3.  Create a Makefile that automatically regenerates results and plots whenever a new data file is added to the project.
+4.  Add error handling and logging to the scripts so that they can be put into production.
+5.  Use style-checking tools to ensure the Python code is well written, and then release it as an installable package.
+6.  Adopt a license, a code of conduct, and efficient decision-making processes to ensure the project runs smoothly.
+7.  Publish results with DOIs, ORCIDs, and other metadata in a fully-reproducible way.
+
+**4. The minimally-qualified reader (MQR)**
+
+*Amira Khan*
+completed a master's in library science five years ago
+and has since worked for a small aid organization.
+She did some statistics during her degree,
+and has learned some R and Python by doing data science courses online,
+but has no formal training in programming.
+Amira would like to tidy up the scripts, data sets, and reports she has created
+in order to share them with her colleagues.
+These lessons will show her how to do this and what "done" looks like.
+
+*Jun Hsu*
+completed a 4-month data science bootcamp last year after doing a PhD in Geology
+and now works for a company that does forensic audits.
+He uses a variety of machine learning and visualization packages,
+and would now like to turn some of his own work into an open source project.
+This book will show him how such a project should be organized
+and how to encourage people to contribute to it.
+
+*Sami Virtanen*
+became a competent programmer during a bachelor's degree in applied math
+and was then hired by the university's research computing center.
+The kinds of applications they are being asked to support
+have shifted from fluid dynamics to data analysis;
+this guide will teach them how to build and run data pipelines
+so that they can pass those skills on to their users.
+
+**5. Q&A**
+
+*What are the three or four most commonly-asked questions about this technology?*
+
+We wrote this book in part because the people who need it don't even know what questions to ask.
+"How do I write a general rule in a Makefile?" presupposes that you know that Make exists.
+Similarly,
+people don't ask, "How should a small group run a meeting?"
+because it doesn't occur to them that running meetings is an improvable skill.
+
+*What are others interested in this topic asking in forums?*
+
+All of our chapters are driven by questions we have answered (some of them many times)
+in workshops or in discussions with colleagues.
+Our answers for technical subjects frequently draw on Stack Overflow,
+particularly for topics like continuous integration and packaging;
+our answers for community management and project leadership draw on extensive experience in open source.
+
+**6. Tell us about the competition and the ecosystem.**
+
+*What other books are available on this topic?*
```

Comment:
There's [A Tour of Data Science](https://www.routledge.com/A-Tour-of-Data-Science-Learn-R-and-Python-in-Parallel/Zhang/p/book/9780367895860), which from what I  can tell is based on an openly available resource called [Another Book on Data Science](https://www.anotherbookondatascience.com/index.html).

Reply:
(A Tour of Data Science is set to be published in December 2020)

Reply:
:+1: